.github/workflows/main.yml

```diff
@@ -120,9 +122,9 @@ jobs:
           persist-credentials: false
 
       - name: Set up Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4.3.0
```

Comment:
Forgive me but what is this semver tag pointing at?

Reply:
I believe it's this https://github.com/actions/setup-python we are updating to take into account that we are using 3.11 now was recently causing some errors

Reply:
Oh I  get it now! Thanks for the quick education.